ci-operator/config/openshift-priv/cluster-network-operator/openshift-priv-cluster-network-operator-master.yaml

```diff
@@ -206,6 +206,40 @@ tests:
       NETWORK_TYPE: OVNKubernetes
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-aws-live-migration-sdn-ovn
+  optional: true
```

Comment:
optional:true? do we want to make it always_run:false? assuming we want it optional because it's not expected to pass at a high rate? I'd like to keep those kinds of jobs from running automatically and we should dedicate some time to bring the pass rate up and then make it required.

Reply:
I need data to find out how reliable it is and thats why I want it to run for a period of time. We have a jira card to review the results and make it required once it has proven itself.

Reply:
I guess an alternative would be to keep retesting here until we see its stable and just make it required.